apps/federatedfilesharing/lib/OCM/CloudFederationProviderFiles.php

```diff
@@ -167,7 +176,7 @@ public function shareReceived(ICloudFederationShare $share) {
 					$this->notifyAboutNewShare($shareWith, $shareId, $ownerFederatedId, $sharedByFederatedId, $name, $ownerDisplayName);
 
 					// If auto-accept is enabled, accept the share
-					if ($this->federatedShareProvider->isFederatedTrustedShareAutoAccept() && $this->trustedServers->isTrustedServer($remote)) {
+					if ($this->federatedShareProvider->isFederatedTrustedShareAutoAccept() && $trustedServers?->isTrustedServer($remote)) {
```

Comment:
nit

```suggestion
					if ($this->federatedShareProvider->isFederatedTrustedShareAutoAccept() && $trustedServers?->isTrustedServer($remote) === true) {
```

null is falsy, so the current code will already work. This just makes the condition more explicit